packages/ethersjs-provider/src/EthersProvider.ts

```diff
@@ -113,7 +114,8 @@ export class EthersProvider extends ethers.providers.BaseProvider
     let gasEstimate = await super.estimateGas(transaction);
     if (this.gasEstimateIncreasePercentage) {
       gasEstimate = gasEstimate.add(
-        gasEstimate.div(this.gasEstimateIncreasePercentage)
+        gasEstimate.div(
+          new ethers.utils.BigNumber(100).div(this.gasEstimateIncreasePercentage))
```

Comment:
There was a bug in how we calculated percentages that wasn't apparent because the value used was 10.